docker-compose-rule-core/src/test/resources/log4j2.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+  ~ Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<Configuration monitorInterval="5">
+    <Appenders>
+        <RollingFile name="LOG" fileName="logs/docker-compose-rule.log" filePattern="logs/docker-compose-rule-%d{yyyy-MM-dd}.log.gz">
+            <PatternLayout pattern="%d{ISO8601} %-5p - %t %c - %m%n"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy interval="1"/>
+            </Policies>
+            <DefaultRolloverStrategy max="30"/>
+        </RollingFile>
+
+        <Console name="STDOUT" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{ISO8601} %-5p - %t %c - %m%n"/>
+        </Console>
+    </Appenders>
+    <Loggers>
+        <Logger name="com.palantir" level="debug" additivity="false">
+            <AppenderRef ref="STDOUT"/>
+            <AppenderRef ref="LOG"/>
+        </Logger>
+
+        <Logger name="com.palantir.dia" level="debug" additivity="false">
+            <AppenderRef ref="STDOUT"/>
+            <AppenderRef ref="LOG"/>
+        </Logger>
+
+        <Logger name="org.apache.camel" level="debug" additivity="false">
+            <AppenderRef ref="STDOUT"/>
+            <AppenderRef ref="LOG"/>
+        </Logger>
```

Comment:
We can remove the `com.palantir.dia` and `org.apache.camel` blocks, remove appenders and `additivity` from the `com.palantir` block in order to get the same output with a more concise configuration.

Reply:
I just decided to split the logging changes into another PR: https://github.com/palantir/docker-compose-rule/pull/289